app/src/main/java/com/sopt/now/compose/feature/signin/SignInScreen.kt

```diff
@@ -0,0 +1,219 @@
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    LaunchedEffect(true) {
+        navController.previousBackStackEntry?.savedStateHandle?.run {
+            val user = get<User>("user") ?: User()
+            viewModel.setInfo(user)
+        }
+    }
+
+    SignInScreen(
+        snackBarHostState,
+        id = state.id,
+        pw = state.pw,
+        signInBtnClicked = {
+            scope.launch {
+                viewModel.signInBtnClicked()
+            }
+        },
+        signUpBtnClicked = {
+            scope.launch {
+                viewModel.signUpBtnClicked()
+            }
+        },
+        fetchId = { viewModel.fetchId(it) },
+        fetchPw = { viewModel.fetchPw(it) }
+    )
+}
+
+@Composable
+fun SignInScreen(
+    snackBarHostState: SnackbarHostState,
+    id: String,
+    pw: String,
+    signInBtnClicked: () -> Unit,
+    signUpBtnClicked: () -> Unit,
+    fetchId: (String) -> Unit,
+    fetchPw: (String) -> Unit
+) {
+    Scaffold(
+        modifier = Modifier.addFocusCleaner(LocalFocusManager.current),
+        snackbarHost = {
+            SnackbarHost(hostState = snackBarHostState)
+        },
+        topBar = {
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(40.dp),
+                contentAlignment = Alignment.Center
+            ) {
+                Text(
+                    text = stringResource(id = R.string.sign_in_title),
+                    fontSize = 30.sp,
+                    fontWeight = FontWeight.ExtraBold
+                )
+            }
+        },
+        bottomBar = {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 20.dp)
+                    .padding(bottom = 30.dp),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Button(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    onClick = signInBtnClicked
+                ) {
+                    Text(text = stringResource(id = R.string.sign_in_title))
+                }
+                Text(
+                    modifier = Modifier.noRippleClickable {
+                        signUpBtnClicked()
+                    },
+                    text = stringResource(id = R.string.sign_up_btn),
+                    fontSize = 12.sp,
+                    fontWeight = FontWeight.Thin,
+                    color = Color.LightGray
+                )
+            }
+        }
+    ) { paddingValues ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(
+                    top = paddingValues.calculateTopPadding(),
+                    bottom = paddingValues.calculateBottomPadding(),
+                    start = 20.dp,
+                    end = 20.dp
+                ),
```

Comment:
```suggestion
                .padding(paddingValues),
                .padding(start = 20.dp, end = 20.dp)
                
```
로 쓸 수 있지 않을 까 싶습니당 혹시 paddingValues에 start, end도 이미 padding이 들어가있다면 죄송합니다 ㅋㅋ

Reply:
안들어있었습죠 ㅋㅋㅋ 수정했습니다!!

근데 하나 다른 관점이 있습니다.
수정을 권유해주신 방법에 비해서 제가 작성한 방법의 장점이 있다고 생각합니다.
top과 bottom을 명시해뒀기 때문에 상하단에 padding이 있다는 것을 한눈에 파악할 수 있습니다.
그래서 제가 한 방법도 나쁘진 않은거 같은데 어떻게 생각하시나요?

어차피 Surface내부에 있는거라 상관이 없나...??
어렵네요